docs/user/addons.rst

```diff
@@ -46,8 +46,323 @@ Individual configuration options for each addon are available in :guilabel:`Sett
 Addons data and customization
 -----------------------------
 
-If you'd like to do a custom integration with the data used to render Addons,
-you can learn more about this in our :ref:`flyout-menu:custom event integration` docs.
+Addons can be customized using CSS variables and the data used by Addons can be accessed using a custom event.
+
+CSS Variable Customization
+~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Addons use CSS custom properties (`CSS variables <https://developer.mozilla.org/en-US/docs/Web/CSS/--*>`_) to allow for easy customization.
+To customize addons, add CSS variable definitions to your theme's CSS:
+
+.. code-block:: css
+
+    :root {
+        /* Reduce Read the Docs' flyout font a little bit */
+        --readthedocs-flyout-font-size: 0.7rem;
```

Comment:
I left a comment about if users should use `rem` or `px` here in https://github.com/readthedocs/addons/issues/488#issuecomment-2586638456 because it's not 100% clear to me yet.

Reply:
`rem` is better as it scales with what the _reader user sets as a font size_. Pixel sizing values won't always scale for these users. `rem` is used over `em` to avoid nested elements increasing font-size.